app/api/internal/internal.rb

```diff
@@ -31,6 +32,20 @@ class Internal < Sinatra::Base
           parse_json_params params
         end
 
+        before do
```

Comment:
This is just temporary to test it properly from https://editor-next.swagger.io/ 
(just to avoid running swagger-ui locally)

In fact, I was not going to commit it, it was an accident. I'll remove later.